app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    summa = sum(split_integer(17, 4))
+    assert summa == 17
```

Comment:
It's not necessary to create a variable for the sum of the result of the function `split_integer`. You can directly compare with the expected result in the `assert` statement. This is a code style issue as per the checklist point 1.